client/tests/e2e/update-description.spec.js

```diff
@@ -0,0 +1,47 @@
+var Etcd = require('node-etcd'),
+    etcd = new Etcd();
+
+describe('Update a feature description', function () {
+    var protractorInstance = protractor.getInstance();
+
+    var descCss = '.editable';
+    var editableInputCss = '.editable-input';
+
+    var removeAllFeatures = function(done){
+        etcd.rmdir('v1', { recursive: true }, function(){
+            done();
+        });
+    };
+
+    beforeEach(function(done) {
+        removeAllFeatures(function() {
+            etcd.set('v1/metadata/TestApp/githubRepoUrl', 'https://github.com/opentable/hobknob', function () {
+                etcd.set('v1/toggles/TestApp/TestFeature', false, function () {
+                    etcd.set('v1/metadata/TestApp/descriptions/TestFeature', 'text', function (){
+                        browser.get('/#!/applications/TestApp/TestFeature');
+                        done();
+                    });
+                });
+            });
+        });
+    });
+
+    afterEach(function(done){
+        removeAllFeatures(done);
+    });
+
+    var getDesc = function(){
+        return element(by.css(descCss));
+    };
+
+    var expectDescToHaveText = function(state){
```

Comment:
The state param here isn't doing anything.. I think you meant to pass the expected value in here?